browser/brave_federated_learning/sources.gni

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+brave_browser_brave_federated_learning_sources = [
```

Comment:
why is this going in sources instead of a BUILD.gn? Is there a circular dep somewhere?